src/v2/Apps/Fair/FairSubApp.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react"
+import { AppContainer } from "v2/Apps/Components/AppContainer"
+import { createFragmentContainer, graphql } from "react-relay"
+import { FairSubApp_fair } from "v2/__generated__/FairSubApp_fair.graphql"
+import { Separator } from "@artsy/palette"
+import { HorizontalPadding } from "v2/Apps/Components/HorizontalPadding"
+import { Footer } from "v2/Components/Footer"
+import { ErrorPage } from "v2/Components/ErrorPage"
+import { BackLink } from "v2/Components/Links/BackLink"
+
+interface FairAppProps {
+  fair: FairSubApp_fair
+}
+
+const FairApp: React.FC<FairAppProps> = ({ children, fair }) => {
+  if (!fair) return <ErrorPage code={404} />
+
+  return (
+    <>
```

Comment:
This fragment in place as a placeholder for when header tags will be inserted?

Reply:
Oh, I had just copied what was in the original `FairApp`, but let's go with that reason, sure :)